plugin/db/clickhouse/sync.go

```diff
@@ -60,19 +60,25 @@ func (driver *Driver) SyncInstance(ctx context.Context) (*db.InstanceMeta, error
 }
 
 // SyncSchema syncs the schema.
-func (driver *Driver) SyncSchema(ctx context.Context) ([]*db.Schema, error) {
+func (driver *Driver) SyncSchema(ctx context.Context, databaseList ...string) ([]*db.Schema, error) {
 	excludedDatabaseList := []string{
 		// Skip our internal "bytebase" database
 		"'bytebase'",
 	}
-
+	var includeDatabaseList []string
 	// Skip all system databases
 	for k := range systemDatabases {
 		excludedDatabaseList = append(excludedDatabaseList, fmt.Sprintf("'%s'", k))
 	}
+	for _, k := range databaseList {
+		includeDatabaseList = append(includeDatabaseList, fmt.Sprintf("'%s'", k))
```

Comment:
do we need to lowercase here as well?

Reply:
Yes, done. Also updated the mysql/sync.go